covid19-etl/utils/metadata_helper.py

```diff
@@ -98,8 +100,14 @@ def get_str_latest_submitted_date_idph(self):
         }"""
         variables = {"filter": {"=": {"project_id": self.project_id}}}
         query_res = self.query_guppy(query_string, variables)
-        loc = query_res["data"]["location"][0]
-        return loc["date"]
+        if "data" not in query_res or "location" not in query_res["data"]:
+            raise Exception(
+                "Did not receive any data from Guppy. Is the token expired?"
```

Comment:
also print `query_string`, `variables` and `query_res` so we have useful debug information in the logs